### ISSUE TITLE
Cells Lacking the RB1 Tumor Suppressor Gene Are Hyperdependent on Aurora B Kinase for Survival

### ISSUE THREAD

**Issue by pnrobinson** [monarch-initiative/IDGKG-gen#133]
https://pubmed.ncbi.nlm.nih.gov/30373918/

**Comment by pnrobinson** [comment]
To identify synthetic lethal interactors with RB1 in SCLC, we first infected two RB1−/− SCLC cell lines (NCI-H82 and NCI-H69) with a lentivirus that expresses RB1 in the presence of doxycycline (DOX-On RB1) or with the corresponding empty vector (DOX-On EV). Given that RB1 reexpression had no gross effect on cell proliferation in NCI-H82 cells, we used the DOX-On RB1 NCI-H82 cells to perform a RB1 synthetic lethal screen using CRISPR/Cas9-based gene editing. 
Using this strategy, we also identified 104 genes that were synthetic lethal with RB1, based on sgRNA depletion in the RB1-deficient cells (DOX-On RB1; no DOX) compared with the RB1-proficient cells (DOX-On RB1; +DOX), using a P value cutoff of <0.05 (Fig. 1G; Supplementary Tables S2 and S4).
